### PR TITLE
chore: polish kb14-kb20 files

### PR DESCRIPTION
**Pull request by Meng-Heng**
I thought I had git rebase under control but there are files in this commit too. I'm not sure if it's part of the rebase or not.

@mcdurdin, is this how it's supposed to be when do `git rebase upstream/staging` success?

The markdown table does not appear to have any border in KB0015, even when I pasted a table from a different file that displayed the correct table.

### CODE REVIEW

**Comment by mcdurdin** [comment]
I changed the base of this PR to `master`, and then merged master in. This cleaned up the files that have been changed, although the commit history is a little bit chaotic. It should be fine!

File: knowledge-base/kb0015.md [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
-
-</table>
+# What is the difference between a licence, registration key, account, activation code, etc?
+
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+
+| Item (Looks like) | Description |
+|-----------|-----------|
+|| |
```
Comment (Meng-Heng) [inline knowledge-base/kb0015.md:8]:
```suggestion
| Item (Looks like)| Description|
|    ---------      |   -------- |
|                   |            |

```

File: knowledge-base/kb0018.md [hunk 2]
```diff
@@ -1,29 +1,22 @@
...
+
+By default, Yahoo Messenger does not accept Unicode input from Keyman Desktop or from Windows keyboards.
+
+## Workaround
+
+The Keyman Desktop RichEdit addin can be configured to work with Yahoo Messenger's Input Window.  This will allow Keyman Desktop Unicode keyboards to work within Yahoo Messenger.  However, Windows Unicode keyboards will not work.
+
+- Make sure that the RichEdit addin is installed in Keyman Configuration.  If it is not installed, then you should re-run the Keyman Installer and select the RichEdit addin for installation.
+- inst_km_yim.exe is no longer available for download.
+- Close down Keyman Desktop and restart it.
```
Comment (Meng-Heng) [inline knowledge-base/kb0018.md:13]:
```suggestion
1. Make sure that the RichEdit addin is installed in Keyman Configuration.  If it is not installed, then you should re-run the Keyman Installer and select the RichEdit addin for installation.
2. inst_km_yim.exe is no longer available for download.
3. Close down Keyman Desktop and restart it.

```

**Review by Meng-Heng — COMMENTED** [review]
Applying the right markdown syntax of `<ol>`

File: knowledge-base/kb0015.md [hunk 3]
```diff
@@ -1,22 +1,13 @@
...
-</table>
+# What is the difference between a licence, registration key, account, activation code, etc?
+
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+
+| Item (Looks like)| Description|
+|    ---------      |   -------- |
+|                   |            |
+
```
Comment (darcywong00) [inline knowledge-base/kb0015.md:9]: I think this extra line breaks up the table formatting
```suggestion
```

File: knowledge-base/kb0016.md [hunk 4]
```diff
@@ -1,26 +1,20 @@
...
-<p>The Unicode Character Database is used only within Keyman Developer Professional and during the evaluation period for Keyman Developer.</p>
-
-## Applies to:
- * Keyman Developer Professional 6.0
- * Keyman Developer Professional 6.2
- * Keyman Developer Standard  6.0
- * Keyman Developer Standard 6.2
+# HOWTO: Fix 'EOleException' or 'EAccessViolation' crash when starting Keyman Developer
+
+###*NOT*: This archived documentation has not been updated recently and may contain information that is no longer relevant
```
Comment (darcywong00) [inline knowledge-base/kb0016.md:3]:
```suggestion
### NOTE: This archived documentation has not been updated recently and may contain information that is no longer relevant
```

**Comment by Meng-Heng** [comment]
All is good, @darcywong00.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**